charts/portworx/values.yaml

```diff
@@ -2,18 +2,20 @@
 
 deploymentType: oci                     # accepts "oci" or "docker"
 imageType: none                         #
-imageVersion: 2.0.3.8                   # Version of the PX Image.
+imageVersion: 2.1.2                    # Version of the PX Image.
```

Comment:
@harsh-px Should we change this to 2.1.3?

Reply:
Yes